python/src/opendp/polars.py

```diff
@@ -6,13 +28,17 @@
 
     @pl.api.register_expr_namespace("dp")
     class DPExpr(object):
+        '''
+        This class is not used directly by users:
+        Instead its methods are registered under the ``dp`` namespace of Polars expressions.
+        '''
```

Comment:
Actually, this class could be used directly by users, to get mypy typing annotations. For example: `DPExpr(pl.len()).noise()`.